openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1342,6 +1342,9 @@ This specification defines the following Credential Issuer Metadata:
 * `display`: OPTIONAL. Array of objects, where each object contains display properties of a Credential Issuer for a certain language. Below is a non-exhaustive list of valid parameters that MAY be included:
   * `name`: OPTIONAL. String value of a display name for the Credential Issuer.
   * `locale`: OPTIONAL. String value that identifies the language of this object represented as a language tag taken from values defined in BCP47 [@!RFC5646]. There MUST be only one object for each language identifier.
+  * `logo`: OPTIONAL. A JSON object with information about the logo of the Credential Issuer with a following non-exhaustive list of parameters that MAY be included:
+    * `uri`: OPTIONAL. String value that contains a URI where the Wallet can obtain a logo of the Credential from the Credential Issuer. Wallet needs to determine the scheme, since the URI value could use `https:` scheme, `data:` scheme, etc.
```

Comment:
```suggestion
    * `uri`: REQUIRED when the `logo` object is present. String value that contains a URI where the Wallet can obtain a logo of the Credential from the Credential Issuer. Wallet needs to determine the scheme, since the URI value could use `https:` scheme, `data:` scheme, etc.
```

If the logo object is present but doesn't contain any properties (like uri), it would indeed result in "logo": {}. This would be nonsensical because the logo object is supposed to contain at least the uri property when it is present. Therefore, it's important to specify that uri is required when logo is present to avoid such situations.

Reply:
we should discuss this. `logo` object is optional. so the issuer can avoid "logo": {} by not including `logo` parameter at all.

Reply:
I think it's also about setting clear expectations for what we expect issuers to do, as well as making it clear what situations we might end up needing to conformance test that wallets correctly parse. So I think it's good to make it clear that we don't expect issuers to use `"logo": {}` if that is our expectation.

It would probably be clearer if the logo object was defined in a new section. Then `uri` could I think just be marked as 'REQUIRED' and I think it would be clear that means "if the logo object is present".

Reply:
I don't think a separate section is necessary. I changed `uri` parameter to `required`. it made more sense from the expectations perspective and from the convention of the spec perspective.